adoc/Harbor-Private-Registry-Installation.adoc

```diff
@@ -0,0 +1,804 @@
+          resources:
+            requests:
+              storage: 10Gi
+  auth:
+    secretPath: redis-auth
+----
++
+At a minimum, the configuration options below should be customized as desired.
+For more configuration options, the link:https://github.com/spotahome/redis-operator#usage[Redis Operator Documentation] may be used.
++
+--
+[lowerroman]
+. Number of replicas for sentinel and Redis. Use a number higher than 2 for a highly-available installation
+. The storage size for the `persistentVolumeClaim`
+--
++
+The `RedisFailover` configuration can be deployed using `kubectl`.
+For example, assuming `redis.yaml` is the file containing the `RedisFailover` object properties, the command would be:
++
+[source,bash]
+----
+kubectl create -n registry -f redis.yaml
+----
+... Wait until all the objects are prepared, specifically `rfs-harbor-redis`, the Sentinel deployment:
++
+[source,bash]
+----
+> kubectl get deployments -l app.kubernetes.io/component=sentinel -n registry
+NAME                READY   UP-TO-DATE   AVAILABLE   AGE
+rfs-harbor-redis    3/3     3            3           7m57s
+----
+
+... Configure {spr} to be connected to the external Redis
++
+Extend the `harbor-values.yaml` file with the configuration specified below.
++
+.harbor-values.yaml
+[source,yaml]
+----
+redis:
+  type: external
+  external:
+    addr: rfs-harbor-redis:26379
+    sentinelMasterSet: mymaster
+    password: securepassword // <1>
+----
+<1> Replace the value for password key with the password prepared in <<install-redis-operator>>.
+
+. (Optional) External Redis Setup
++
+[#install-external-redis]
++
+An external Redis is recommended to deploy {spr} in a fully highly-available and scalable setup.
+When deployed in AKS or EKS, as an alternative to using the Redis Operator, {spr} may instead be connected to a managed Redis instance running in public cloud.
+This section assumes a managed Redis instance has already been setup, either in Azure or AWS, as covered in the External Redis requirements section.
+
+.. Connect to an Azure Cache for Redis instance
++
+Add the following section to the `harbor-values.yaml` file and fill it with information reflecting the Azure Cache for Redis instance previously prepared.
+As mentioned above in the <<requirements-redis-azure>>, the address will have the form of `<azure-redis-cache>.redis.cache.windows.net`.
++
+.harbor-values.yaml
+[source,yaml]
+----
+redis:
+  type: external
+  external:
+    addr for redis: <host_redis>:<port_redis>
+    addr: "192.168.0.2:6379"
+----
+// TODO AI (Dirk Mueller) needs to be reviewed
+.. Connect to an Amazon ElastiCache Redis service
++
+Add the following section to the `harbor-values.yaml` file and fill it with information reflecting the Amazon ElastiCache Redis instance previously prepared:
++
+.harbor-values.yaml
+[source,yaml]
+----
+redis:
+  type: external
+  external:
+    addr for redis: <host_redis>:<port_redis>
+    addr: "192.168.0.2:6379"
+    password: securepassword
```

Comment:
There should be a note here that the password is the value used when preparing Redis in AWS...

Reply:
That is not something we described in this document, right?

Reply:
Yeah, that's part of (still missing) _Connect to an Amazon ElastiCache Redis service_ section. I think you can actually create such (empty) section after _Azure Cache for Redis_ one.